contracts/wrapped-assets/shiba/WrappedShibaSwapLp.sol

```diff
@@ -212,10 +226,9 @@ contract WrappedShibaSwapLp is IWrappedAsset, Auth2, ERC20, ReentrancyGuard {
      * @param _boneLocker BoneLocker to claim, pass zero address to claim from current
      * @param _maxBoneLockerRewardsAtOneClaim max amount of rewards items to claim from BoneLocker, pass 0 to claim all rewards
      */
-    function claimRewardFromBoneLocker(address _user, IBoneLocker _boneLocker, uint256 _maxBoneLockerRewardsAtOneClaim) public nonReentrant {
-        WSSLPUserProxy userProxy = _requireUserProxy(_user);
-
-        userProxy.claimRewardFromBoneLocker(_boneLocker, _maxBoneLockerRewardsAtOneClaim, feeReceiver, feePercent);
+    function claimRewardFromBoneLocker(IBoneLocker _boneLocker, uint256 _maxBoneLockerRewardsAtOneClaim) public nonReentrant {
+        WSSLPUserProxy userProxy = _requireUserProxy(msg.sender);
```

Comment:
note: also, now only owner coud call this in case of unexpected future updates of bone locker